From nobody Mon Sep 17 00:00:00 2001
From: Cliff Brake <cbrake@happy.dev.bec-systems.com>
Date: Tue Apr 3 22:17:25 2007 -0400
Subject: [PATCH] mmcsd_no_scr_check r0

mmc_no_sd check

---

 drivers/mmc/mmc.c |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

base b6144d28fde6225f49d1d56e49b333b7ff11d8dd
last 8139d6c8b46a7a041094753eced2d46e006d93ab
diff --git a/drivers/mmc/mmc.c b/drivers/mmc/mmc.c
index 6f2a282e2b9759c0511cf5501a0bd4e9bd501e3f..b0c830c65a27f5579a2d4f67345e4d9bd1f0fe24 100644
--- a/drivers/mmc/mmc.c
+++ b/drivers/mmc/mmc.c
@@ -670,9 +670,11 @@ static void mmc_decode_scr(struct mmc_card *card)
 
 	scr_struct = UNSTUFF_BITS(resp, 60, 4);
 	if (scr_struct != 0) {
-		printk("%s: unrecognised SCR structure version %d\n",
+		printk("%s: Warning - unrecognised SCR structure version %d\n",
 			mmc_hostname(card->host), scr_struct);
-		mmc_card_set_bad(card);
+		//mmc_card_set_bad(card);
+		scr->bus_widths = 0;
+		scr->sda_vsn = 0;
 		return;
 	}
 
-- 
1.4.4.4

